packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -98,4 +99,19 @@ describe('quantic-tab', () => {
       Expect.activeTabContains(tabs.knowledge.label);
     });
   });
+
+  describe('when using accessibility', () => {
+    it('should be accessible to keyboard', () => {
+      loadFromUrlHash({}, `tab=${tabs.case.label}`);
+
+      Actions.keyPressButton(tabs.all.label);
```

Comment:
```suggestion
      Actions.pressSpaceOnTarget(tabs.all.label);
```

Reply:
```suggestion
      Actions.pressSpaceOnElement(tabs.all.label);
```

Reply:
One of those options would be much clearer to create as a specific function.

A function called `keyPressButton` with a default value that presses space is not clear what it does really with its name (why does it need a target) and also nothing is letting the user know that not passing a 2nd parameter means this will press space by default.